public/static/docs/command-reference/diff.md

```diff
@@ -99,7 +99,7 @@ diff for 'data/data.xml'
 added file with size 37.9 MB
 ```
 
-## Example: Specific targets across Git references
+## Example: Specific targets across Git revisions
```

Comment:
so, we have a total mix of revisions and references in one file? considering that we moving to revisions - let's use revision and commit as a user friendly option

Reply:
Haha yeah that was strange. I removed the term "revision" completely from here (and started using "commit" more in 43120ad1. "reference" is still there given the `a/b_ref` options' name.